table.test.js

```diff
@@ -36,23 +36,87 @@ var browsers = {
+var demoEndpoint = 'http://localhost:8081/components/d2l-table/demo/index.html';
+
+polymerTests(browsers, function(test, ctx) {
+	var hasShadow = ctx.driver.executeScript('return Boolean(Element.prototype.createShadowRoot)').booleanValue();
+
+	// https://github.com/webcomponents/shadydom/issues/168
+	if (!hasShadow) {
+		test('d2l-table', {
+			endpoint: mainlineEndpoint + '?wc-shadydom&wc-shimcssproperties',
+			spec: 'test/acceptance/table.gspec',
+			tags: ['mainline', 'shady', 'shim-css', 'ltr']
+		});
+
+		test('d2l-table-rtl', {
+			endpoint: mainlineEndpoint + '?wc-shadydom&dir=rtl&wc-shimcssproperties',
+			spec: 'test/acceptance/table.rtl.gspec',
+			tags: ['mainline', 'shady', 'shim-css', 'rtl']
+		});
+
+		test('d2l-table-css', {
+			endpoint: mainlineEndpoint + '?wc-shadydom&useNativeCSSProperties=true',
+			spec: 'test/acceptance/table.gspec',
+			tags: ['mainline', 'shady', 'native-css', 'ltr']
+		});
+
+		test('d2l-table-css-rtl', {
+			endpoint: mainlineEndpoint + '?wc-shadydom&dir=rtl&useNativeCSSProperties=true',
+			spec: 'test/acceptance/table.rtl.gspec',
+			tags: ['mainline', 'shady', 'native-css', 'rtl']
+		});
+	}
+
+	test.shadow('d2l-table-shadow', {
+		endpoint: mainlineEndpoint + '?dom=shadow&useNativeCSSProperties=true',
+		spec: 'test/acceptance/table.shadow.gspec',
+		tags: ['mainline', 'shadow', 'native-css', 'ltr']
+	});
+
+	/*
+	// This spec fails because the icon mirroring is broken in Chrome's ShadowDOM
+	test.shadow('d2l-table-rtl-shadow', {
+		endpoint: endpoint + '?dir=rtl&dom=shadow',
+		spec: 'test/acceptance/table.rtl.shadow.gspec'
+	});*/
 
-polymerTests(browsers, function(test) {
 	test('d2l-table', {
-		endpoint: endpoint,
-		spec: 'test/acceptance/table.gspec'
+		endpoint: xEndpoint + '?wc-shadydom',
+		spec: 'test/acceptance/table.gspec',
+		tags: ['1.x', 'shady', 'shim-css', 'ltr']
 	});
 
 	test('d2l-table-rtl', {
-		endpoint: endpoint + '?dir=rtl',
-		spec: 'test/acceptance/table.rtl.gspec'
+		endpoint: xEndpoint + '?wc-shadydom&dir=rtl',
+		spec: 'test/acceptance/table.rtl.gspec',
+		tags: ['1.x', 'shady', 'shim-css', 'rtl']
 	});
 
 	test.shadow('d2l-table-shadow', {
-		endpoint: endpoint + '?dom=shadow',
-		spec: 'test/acceptance/table.shadow.gspec'
+		endpoint: xEndpoint + '?dom=shadow',
+		spec: 'test/acceptance/table.shadow.gspec',
+		tags: ['1.x', 'shadow', 'shim-css', 'ltr']
+	});
+
+	test('d2l-table-css', {
+		endpoint: xEndpoint + '?wc-shadydom&useNativeCSSProperties=true',
+		spec: 'test/acceptance/table.gspec',
+		tags: ['1.x', 'shady', 'native-css', 'ltr']
+	});
+
+	test('d2l-table-css-rtl', {
+		endpoint: xEndpoint + '?wc-shadydom&dir=rtl&useNativeCSSProperties=true',
+		spec: 'test/acceptance/table.rtl.gspec',
+		tags: ['1.x', 'shady', 'native-css', 'rtl']
+	});
+
+	test.shadow('d2l-table-css-shadow', {
+		endpoint: xEndpoint + '?dom=shadow&useNativeCSSProperties=true',
+		spec: 'test/acceptance/table.shadow.gspec',
+		tags: ['1.x', 'shadow', 'native-css', 'ltr']
 	});
 
```

Comment:
Looks like it's a known issue: https://github.com/webcomponents/shadycss/issues/91